tests/samples/expected/nickel.ncl

```diff
@@ -4,14 +4,19 @@
 {
   arrays_arrays = let my_array_lib =
     {
-      map : forall a b. (a -> b) -> Array a -> Array b = fun f arr =>
+      map : forall a b. (a -> b) -> Array a -> Array b
+        | doc m%"
+          Here is some documentation
+          "%
```

Comment:
I think we'd usually like to indent the documentation once further, so
```suggestion
        | doc m%"
            Here is some documentation
          "%
```

Reply:
No luck, I'm afraid. If I try to format a multiline string as if it were just another syntactic construction, then strange things happen:
1. If I force a newline after the multiline string start token, it breaks idempotency because it keeps trying to insert a new line on each run.
2. The indentation -- which was the hope that formatting would address -- appears to be unaffected.

As such, I think the safest option for now would treat Nickel strings no differently than strings in any other language; i.e., they shouldn't be formatted and it's up to the input to be aligned manually (which I've done in 6888c0f7d1f0).